nohup: no se tendr√° en cuenta la entrada
Using gpu device 0: GeForce GTX TITAN X
DEBUG:__main__:_loading state: xerox_400_400.py
DEBUG:__main__:_prefix: /home/lvapeab/smt/software/GroundHog/tutorials/models/xerox/es_500_500
DEBUG:__main__:_seqLength: 80
DEBUG:__main__:_input units: [500]
DEBUG:__main__:_dout units: 500
DEBUG:__main__:_hidden units: [500]
DEBUG:__main__:_Shortcut inpout: False
DEBUG:__main__:_NCE: False
DEBUG:__main__:_create forward RNN
DEBUG:__main__:Layer:forward_rnn
DEBUG:__main__:_create_embedding_layers
DEBUG:__main__:_create_recurrent_layer
DEBUG:__main__:_create_emb_out_layers
DEBUG:__main__:_create_emb_state_layer
DEBUG:__main__:Create rec_layer
DEBUG:__main__:_create backward RNN
DEBUG:__main__:Layer:backward_rnn
DEBUG:__main__:_create_embedding_layers
DEBUG:__main__:_create_recurrent_layer
DEBUG:__main__:_create_emb_out_layers
DEBUG:__main__:_create_emb_state_layer
DEBUG:__main__:Create rec_layer
DEBUG:__main__:Create output_layer
DEBUG:__main__:_build train model
DEBUG:groundhog.layers.cost_layers:Get grads
DEBUG:groundhog.layers.cost_layers:Got grads
DEBUG:__main__:_build validation model
DEBUG:__main__:_create_forward_RNN
DEBUG:__main__:Create rec_layer
DEBUG:__main__:_create_forward_RNN
DEBUG:__main__:Create rec_layer
/home/lvapeab/software/anaconda/lib/python2.7/site-packages/theano/scan_module/scan_perform_ext.py:135: RuntimeWarning: numpy.ndarray size changed, may indicate binary incompatibility
  from scan_perform.scan_perform import *
DEBUG:groundhog.datasets.LM_dataset:Not supported
Traceback (most recent call last):
  File "../bLM.py", line 636, in <module>
    jobman(state, None)
  File "../bLM.py", line 619, in jobman
    main.main()
  File "build/bdist.linux-x86_64/egg/groundhog/mainLoop.py", line 305, in main
IndexError: index 200001 is out of bounds for axis 0 with size 200001
